src/css/profile/mobile/common/listview.less

```diff
@@ -105,7 +104,8 @@ tau-expandable {
 		&.ui-li-anchor {
 			display: flex;
 			flex-direction: column;
-			padding: 14 * @px_base 4 * @px_base;
+			padding: 14 * @px_base 0;
+			margin: 0 24 * @px_base;
```

Comment:
I think in current implementation we have to take account width of border line (1px)

Reply:
I'm wondering how reduce this border line. Is possible to use border-image style like in slider widget.
https://github.com/Samsung/TAU/pull/786
Similar to:
```
border-image: url(images/3_Controllers/tw_progress_bar_n.9.png) 5 fill;
border-image-width: 3px;
border-image-repeat: round;
```
but of course different image (with transparency) and different parameters.

Reply:
Do we consider the border width? Why?

Reply:
In 6-2, the text has 24 margin on the left and right.
Do we consider the border line at this case?

Reply:
There is outline stroke. In current implementation this line width are added to margin and in fact elements on list are positioned 25px.

Reply:
You mean the border width of the 'ui-content-area'?
I see.. Okay,, I will try to find how to resolve that issue.

Reply:
I think we can fix border issue with outline.
I will fix it in next PR.